contracts/test/LightClient.t.sol

```diff
@@ -822,93 +820,50 @@ contract LightClient_StateUpdatesTest is LightClientCommonTest {
-        lc.lagOverEscapeHatchThreshold(updates[0] + 2, DELAY_THRESHOLD); //3
+        lc.lagOverEscapeHatchThreshold(block.number + 10, 5); // No updates exist in history
     }
 
-    function test_revertWhenThereIsOnlyOneUpdate() public {
-        uint8 numUpdates = 1;
-
-        uint64[] memory updates = new uint64[](numUpdates);
-        updates[0] = 1;
-
-        LC.StateHistoryCommitment[] memory stateHistoryCommitments =
-            new LC.StateHistoryCommitment[](numUpdates);
-
-        uint64[] memory blockTimestampUpdates = new uint64[](numUpdates);
-        for (uint8 i = 0; i < numUpdates; i++) {
-            blockTimestampUpdates[i] = initialBlockTimestamp + ((i + 1) * 1 days);
-        }
-
-        for (uint256 i = 0; i < updates.length; i++) {
-            stateHistoryCommitments[i] = LC.StateHistoryCommitment({
-                l1BlockHeight: updates[i],
-                l1BlockTimestamp: blockTimestampUpdates[i],
-                hotShotBlockHeight: newState.blockHeight,
-                hotShotBlockCommRoot: newState.blockCommRoot
-            });
-        }
-
-        lc.setStateHistory(stateHistoryCommitments);
+    function test_revertWhenBlockNumberNoStateHistory() public {
+        // note, the state history is updated when the contract is initialized
+        LC.StateHistoryCommitment[] memory emptyStateHistoryCommitments =
+            new LC.StateHistoryCommitment[](0);
+        lc.setStateHistory(emptyStateHistoryCommitments);
 
-        vm.roll(DELAY_THRESHOLD * 3);
-
-        assertEq(lc.getStateHistoryCount(), numUpdates);
+        //assert that there is no state history
+        assertEq(lc.getStateHistoryCount(), 0);
 
         vm.expectRevert(LC.InsufficientSnapshotHistory.selector);
-        lc.lagOverEscapeHatchThreshold(updates[0] + 2, DELAY_THRESHOLD); //3
+        lc.lagOverEscapeHatchThreshold(block.number, 5); // No updates exist in history
     }
 
-    function test_revertWhenBlockRequestedWithinFirstTwoUpdates() public {
+    function test_checkLagWhenDelayThresholdIsOne() public {
         // DELAY_THRESHOLD = 6
-        uint8 numUpdates = 3;
-
-        uint64[] memory updates = new uint64[](numUpdates);
-        updates[0] = 1;
-        updates[1] = updates[0] + DELAY_THRESHOLD / 2; // 4
-        updates[2] = updates[1] + DELAY_THRESHOLD / 2; // 7
+        uint8 numUpdates = 6;
 
-        LC.StateHistoryCommitment[] memory stateHistoryCommitments =
-            new LC.StateHistoryCommitment[](numUpdates);
+        uint64[] memory blockNumberUpdates = new uint64[](numUpdates);
+        blockNumberUpdates[0] = 1;
+        blockNumberUpdates[1] = blockNumberUpdates[0] + DELAY_THRESHOLD / 2; // 4
+        blockNumberUpdates[2] = blockNumberUpdates[1] + DELAY_THRESHOLD / 2; // 7
+        blockNumberUpdates[3] = blockNumberUpdates[2] + DELAY_THRESHOLD + 5; // 18
+        blockNumberUpdates[4] = blockNumberUpdates[3] + DELAY_THRESHOLD / 2; // 21
+        blockNumberUpdates[5] += 1; //22
```

Comment:
Isn't this 0 + 1 = 1?

Reply:
thanks for spotting that, updated here https://github.com/EspressoSystems/espresso-sequencer/commit/3d5ae7aefea737d7611d26f78a02c08144029481#diff-08d922b027356f5ca532486fcea6a356b593fadcc63440be9268b59f2503d5d8R856